server/users.ts

```diff
@@ -530,6 +530,7 @@ export class User extends Chat.MessageContext {
 		return status;
 	}
 	authAtLeast(minAuth: string, room: BasicRoom | null = null) {
```

Comment:
This function, too: it's probably redundant with `Auth#atLeast`, right?

Reply:
It is.